packages/cli/src/commands/hydrogen/dev-vite.ts

```diff
@@ -116,27 +117,37 @@ export async function runDev({
 
   const root = appPath ?? process.cwd();
 
-  const customerAccountPush = await checkMockShopAndByPassTunnel(
+  let appName: string | undefined;
+
+  const cliCommandPromise = getCliCommand(root);
+  const backgroundPromise = checkMockShopAndByPassTunnel(
```

Comment:
UX would like to move the banner in this method to be shown after `enhanceH2Logs` so we may have to re-think this...><...

Reply:
OK, I've split this function in different parts and is now possible:

<img width="735" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/6fae4092-f70a-4bcf-9508-041cb9b19b41">